bedrock/mozorg/templatetags/misc.py

```diff
@@ -599,6 +599,51 @@ def ifeq(a, b, text):
     return jinja2.Markup(text if a == b else '')
 
 
+@library.global_function
+@jinja2.contextfunction
+def app_store_url(ctx, base_url):
```

Comment:
I think it would be nice to simplify how these are called. I think it might be a pain to look up the setting name every time you want to use this and the `play_store_url` helpers. I can see a couple of good options.

You could use these functions as just local python helpers and expose a function per product to the templates. So you'd have `app_store_url_lockwise()`e.g.. The other good option IMO is to use the fact that you've got a good variable naming convention going and use a simple string passed into the function to get the right variable. For example:

```python
def app_store_url(ctx, product):
    base_url = getattr(settings, f'APPLE_APPSTORE_{product.upper()}_LINK')
```

Certainly not required, but should help when you want to use these in future.